ibis/expr/api.py

```diff
@@ -4032,6 +4032,53 @@ def _table_count(self):
     return ops.Count(self, None).to_expr().name('count')
 
 
+def _table_dropna(self, how='any', subset=None):
+    """
+    Remove rows with null values from the table.
+
+    Parameters
+    ----------
+    how : determine whether a row is removed if there is at least one null
```

Comment:
Can you move the description of the parameter below its name, like this:

```
Parameters
----------
how
  determine whether a row is removed if there is at least one null
```

only types should come after the colon.

Reply:
+1 on Examples

Reply:
Fixed the parameters + added examples